src/shared/components/VideoTileBase/VideoTileBase.styles.tsx

```diff
@@ -140,12 +140,6 @@ export const CoverImage = styled.img<CoverImageProps>`
   ${({ darkenImg }) => darkenImg && `filter: brightness(45%);`}
 `
 
-export const CoverNoImage = styled.div`
-  width: 100%;
-  height: 100%;
-  background: linear-gradient(125deg, rgba(16, 18, 20, 1) 30%, rgba(34, 36, 38, 1) 65%, rgba(16, 18, 20, 1) 100%);
-`
```

Comment:
I think we still need this for drafts? We don't want those to keep loading as there's nothing to load